### PR TITLE
do not run tests twice

### PR DESCRIPTION
**Pull request by joernu76**

### CODE REVIEW

**Comment by joernu76** [comment]
@ReimarBauer Do you have an idea, why this does not work? On data-retrieval, this (seems to?) works as expected...

File: .github/workflows/python-flake8.yml [hunk 1]
```diff
@@ -3,7 +3,14 @@
 
 name: flake8
 
-on: [ push, pull_request ]
+on:
+  push:
+    branches:
+    - develop, stable
```
Comment (ReimarBauer) [inline .github/workflows/python-flake8.yml]: from the yaml tutorial this should as list one item per line 
or you need brackets, e.g.  branches: [ develop, stable ]

see 
- https://www.cloudbees.com/blog/yaml-tutorial-everything-you-need-get-started
- https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions

(I guess it searches now the whole string of the line)

File: .github/workflows/testing.yml [hunk 2]
```diff
@@ -1,6 +1,13 @@
 name: Pytest MSS
 
-on: [ push, pull_request ]
+on:
+  push:
+    branches:
+    - develop, stable
```
Comment (ReimarBauer) [inline .github/workflows/testing.yml]: same "develop, stable" may not work

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
use one list item per line.

**Review by ReimarBauer — APPROVED** [review]
thx

**Pull request merged**